demo/return-observable/return-observable.php

```diff
@@ -2,5 +2,5 @@
 
 require_once __DIR__ . '/../bootstrap.php';
 
-$observable = new Rx\Observable\ReturnObservable(42);
+$observable = new Rx\Observable\ReturnObservable(42, \Rx\Scheduler::getDefault());
```

Comment:
The scheduler is no longer optional for Observables and Operators that require a scheduler.